packages/authentication/package.json

```diff
@@ -4,9 +4,10 @@
   "description": "A LoopBack component for authentication support.",
   "scripts": {
     "acceptance": "mocha --opts ../../test/mocha.opts 'test/acceptance/**/*.ts'",
-    "build": "npm run build:lib && npm run build:lib6",
+    "build": "npm run build:lib && npm run build:lib6 && npm run build:apidocs",
```

Comment:
> So, why are we adding this step here in loopback-next?

Let me clarify that. This is what I am asking: why cannot we use the very same mechanism that's already in place to generate API docs from published packages? Is it because we are not publishing typescript sources? If that's the case, then I think it's much better to start publishing typescript sources. It has other benefits besides API docs generation, for example people debugging through LoopBack.next code will be able to step through the original TS code, thanks to sourcemap files we are already publishing.